### ISSUE TITLE
NTR: History of surgery for acute abdominal pain (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7882]
New term request History of surgery for acute abdominal pain (elthat_120821184211):   UMLS: 0455612

**Comment by pnrobinson** [comment]
@MickeySegal
this seems underspecified, since if the surgery happened, we would know why. 
Is History of abdominal surgery sufficient?

**Comment by MickeySegal** [comment]
Yes.  We had made the same change ourselves.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000311
- label: History of abdominal surgery
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: A past medical history that indicates that the patient had a surgical procedure involving the abdomen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7882
- synonyms:
